projects/authentication/docs/user-access-token.md

```diff
@@ -430,9 +419,9 @@ The best way to check if a refresh token is expired is to exchange it for an acc
 
 When the user of your application wants to log out, clear any session data in your application including the user's access token and refresh token. If you have cached the [user's info](#user-info), make sure to also clear that.
 
-Afterward, you should **redirect** the user to the `/logout` URL on the authorization server so the user's session is also cleared there. This redirect will be invisible to the end user, as the authorization server will simply clear the user's session and then redirect back to your application based on a `?returnTo=...` URL parameter that you can specify. Note that you can only use an allowed "logout URL" as value for the `returnTo` URL parameter. (See [Requirements](#requirements))
+Afterward, you should **redirect** the user to the `/realms/uitid/protocol/openid-connect/logout` URL on the authorization server so the user's session is also cleared there. This redirect will be invisible to the end user, as the authorization server will simply clear the user's session and then redirect back to your application based on a `?post_logout_redirect_uris=...` URL parameter that you can specify. Note that you can only use an allowed "logout URL" as value for the `post_logout_redirect_uris` URL parameter. (See [Requirements](#requirements))
 
```

Comment:
Maybe also add a paragraph about **id_token_hint** ?

Reply:
Fair point. It's mentioned in the link behind the link behind `OpenID Connect RP-Initiated Logout` in the following paragraph, but I can make it more explicit.

Reply:
done